packages/component_code_gen/main.py

```diff
@@ -43,6 +43,6 @@
         with open(args.instructions, 'r') as f:
             instructions = f.read()
 
-        result = main(args.type, args.app, instructions, args.tries,
+        result = main(args.type, args.app, instructions, instructions, args.tries,
```

Comment:
If we're passing the same value for both the `instructions` and `prompt` args now, is it worth refactoring to only pass instructions? Or is there still a reason to keep them separate?

Reply:
This was added to the previous way we called the script, by passing a local instructions.md file to it.

I think we can refactor to basically remove everything except for Github Issues. It's the easiest approach on how we interface with the script. Wdyt?